CenterForOpenScience/osf.io
# [Feature] Add pulse modal to institution project settings [OSF-7107]

## Purpose

Adding/removing institutions from a project takes a long time on production. Users need to know something is happening.
## Changes

Add unremovable drop down modal with loading icon until ajax finishes.

![pulse_load](https://user-images.githubusercontent.com/1322421/28837275-29b04c4c-76ba-11e7-9228-4cfd97e226d8.gif)

## Side effects

None known.
## Ticket

https://openscience.atlassian.net/browse/OSF-7107

[#OSF-7107]
## QA Considerations
1. Make sure the modal always shows up and disappears. On staging (and local) it will disappear super fast, making this a little bit tricky. 
2. Make sure it shows up and disappears for all combinations that the add/remove can be done (e.g., node add, node remove, node with components add, node with components remove)
